plugins/connection/aws_ssm.py

```diff
@@ -602,49 +635,55 @@ def start_session(self):
 
         return session
 
-    @_ssm_retry
-    def exec_command(self, cmd, in_data=None, sudoable=True):
-        """run a command on the ssm host"""
-
-        super().exec_command(cmd, in_data=in_data, sudoable=sudoable)
-
-        self._vvv(f"EXEC: {to_text(cmd)}")
-
-        session = self._session
-
-        mark_begin = "".join([random.choice(string.ascii_letters) for i in xrange(self.MARK_LENGTH)])
-        if self.is_windows:
-            mark_start = mark_begin + " $LASTEXITCODE"
-        else:
-            mark_start = mark_begin
-        mark_end = "".join([random.choice(string.ascii_letters) for i in xrange(self.MARK_LENGTH)])
+    def poll_stdout(self, timeout: int = 1000) -> bool:
```

Comment:
From what I understand the function returns the result of .poll(timeout), which is a list of (fd, event) tuples, not necessarily bool. Should the return be `bool(self._polling_obj.poll(timeout)` ?

Reply:
updated